vnext/ReactWindows-UWP.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27310.2036
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29215.179
```

Comment:
@JunielKatarn Does that look ok to you?

Reply:
It does. Because the file format has been the same since about 2010, the only difference will be what VS version icon is displayed.

It's OK to just bump to the latest version available for this header in SLN files.